megamek/src/megamek/common/Entity.java

```diff
@@ -2233,6 +2241,11 @@ public boolean canGoUp(int assumedElevation, Coords assumedPos) {
                 break;
             case VTOL:
                 maxAlt = hex.surface() + 50;
+                // When under a bridge, restrict upward movement
+                if (hex.containsTerrain(Terrains.BRIDGE_ELEV)
+                        && assumedElevation < hex.terrainLevel(Terrains.BRIDGE_ELEV)) {
+                    maxAlt = hex.terrainLevel(Terrains.BRIDGE_ELEV) - height() - 1;   
```

Comment:
If this takes into account `height()` being one less, as below, can you include that note here too?

Reply:
Done.